test/backend/models/people.spec.ts

```diff
@@ -17,11 +18,17 @@ describe("backend/models/people", function () {
 
   it("invalid nameInBanner seperated by too many spaces", async function () {
     const people = People.findByBannerName("Too Many Spaces");
-    return expect(people).to.be.eventually.rejectedWith(BadRequest, "expected nameInBanner to contain only two words, seperated by space");
+    return expect(people).to.be.eventually.rejectedWith(
+      BadRequest,
+      "expected nameInBanner to contain only two words, seperated by space"
+    );
   });
 
   it("invalid nameInBanner with first word more than one character", async function () {
     const people = People.findByBannerName("Long FirstWord");
-    return expect(people).to.be.eventually.rejectedWith(BadRequest, "expected first word in nameInBanner to have a length of 1");
+    return expect(people).to.be.eventually.rejectedWith(
+      BadRequest,
+      "expected first word in nameInBanner to have a length of 1"
+    );
```

Comment:
Does this work? Do you not need await with eventually?

Reply:
if you invoke it without the await, it returns the `Promise` itself, so you hand the `Promise` to the expect, and the eventually handles awaiting the promise for you